### PR TITLE
Revamp Logging Room and Questionnaire Feature

### PR DESCRIPTION
**Pull request by RickyKennedy25**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -91,6 +90,8 @@ const questionnaireResultsQuestionnaireDetail = () => import(/* webpackChunkName
   '@/views/Questionnaire/QuestionnaireResultsQuestionnaireDetail.vue')
 const questionnaireResultsQuestionDetail = () => import(/* webpackChunkName: 'p-questionnaire_results_question_detail' */
   '@/views/Questionnaire/QuestionnaireResultsQuestionDetail.vue')
+const menuMobileQuestionnaire = () => import(/* webpackChunkName: 'p-peer-appraisal' */
```
Comment (jonathan016) [inline src/router/index.js]: Use `/* webpackChunkName: 'p-menu_mobile_questionnaire' */` instead

File: src/router/index.js [hunk 2]
```diff
@@ -1252,16 +1270,55 @@ const router = new Router({
       component: loggingRoom,
       meta: {
         title: 'Logging Rooms',
-        auth: true
+        auth: true,
+        breadcrumb: [
+          { name: 'Home', link: 'feeds' },
+          { name: 'logging Room', link: 'loggingRoom' }
```
Comment (jonathan016) [inline src/router/index.js]: `logging -> Logging` in `name` field

Please also check the rest of this file for this occurrence

File: src/views/LoggingRoom/LogMessage.vue [hunk 3]
```diff
@@ -24,82 +22,80 @@
...
+      }
+
+      &__time {
+        display: flex;
+        align-items: flex-end;
+        font-size: 0.8rem;
+      }
+    }
+
+  /*  .message-card > p {*/
```
Comment (jonathan016) [inline src/views/LoggingRoom/LogMessage.vue]: Maybe remove comment?

File: src/views/LoggingRoom/js/log-message-room.js [hunk 4]
```diff
@@ -54,11 +57,21 @@ export default {
...
+          type: 'is-danger'
+        }
+      })
     },
     submitMessage () {
       loggingRoomApi.createLogMessage(response => {
-        this.$toasted.success('success add log')
+        this.toast({
+          data: {
+            message: 'success add log',
```
Comment (jonathan016) [inline src/views/LoggingRoom/js/log-message-room.js]: `success -> Success`

File: src/views/LoggingRoom/js/logging-room-create.js [hunk 5]
```diff
@@ -64,7 +59,12 @@ export default {
       if (this.titleTemp && this.descriptionTemp && this.members.length) {
         if (!this.isEdit) {
           loggingRoomApi.createLoggingRoom(response => {
-            this.$toasted.success('created')
+            this.toast({
+              data: {
+                message: 'success created logging room',
```
Comment (jonathan016) [inline src/views/LoggingRoom/js/logging-room-create.js]: `success -> Success`, check again on the rest of this file

File: src/views/Questionnaire/js/questionnaire-results-member-detail.js [hunk 6]
```diff
@@ -77,5 +83,16 @@ export default {
       },
       fail: this.errorHandler
     })
+    //
```
Comment (jonathan016) [inline src/views/Questionnaire/js/questionnaire-results-member-detail.js]: Maybe remove comment?

File: src/views/Questionnaire/MyQuestionnaireAppraisee.vue [hunk 7]
```diff
@@ -1,35 +1,56 @@
...
-          ></QuestionnaireParticipantCard>
+          <div class="my-questionnaire-appraisees__content__appraisees-list__appraisee-card-container">
+            <QuestionnaireParticipantCard v-for="appraisee in myListAppraisees"
+                                          :key="appraisee.id"
+                                          :name="appraisee.name"
+                                          :avatar="appraisee.avatar"
+                                          :role="appraisee.role"
+                                          :university="appraisee.university"
+                                          :batch="appraisee.batch.name"
+                                          v-on:click="goToInputQuestionnaireAnswer(appraisee)"
```
Comment (DavidWilliamK) [inline src/views/Questionnaire/MyQuestionnaireAppraisee.vue]: Can be changed `@click` instead of `v-on`

Comment (RickyKennedy25) [reply src/views/Questionnaire/MyQuestionnaireAppraisee.vue]: Resolved, please check again

File: src/views/Questionnaire/js/questionnaire-results-member-detail.js (hunk 6 above)
Comment (RickyKennedy25) [reply src/views/Questionnaire/js/questionnaire-results-member-detail.js]: Resolved, please check again

File: src/views/LoggingRoom/js/logging-room-create.js (hunk 5 above)
Comment (RickyKennedy25) [reply src/views/LoggingRoom/js/logging-room-create.js]: Resolved, please check again

File: src/views/LoggingRoom/js/log-message-room.js (hunk 4 above)
Comment (RickyKennedy25) [reply src/views/LoggingRoom/js/log-message-room.js]: Resolved, please check again

File: src/views/LoggingRoom/LogMessage.vue (hunk 3 above)
Comment (RickyKennedy25) [reply src/views/LoggingRoom/LogMessage.vue]: Resolved, please check again

File: src/router/index.js (hunk 2 above)
Comment (RickyKennedy25) [reply src/router/index.js]: Resolved, please check again.

File: src/router/index.js (hunk 1 above)
Comment (RickyKennedy25) [reply src/router/index.js]: Resolved, please check again.

**Pull request merged**